src/box/memtx_engine.h

```diff
@@ -209,7 +217,8 @@ struct memtx_engine *
 memtx_engine_new(const char *snap_dirname, bool force_recovery,
 		 uint64_t tuple_arena_max_size, uint32_t objsize_min,
 		 bool dontdump, unsigned granularity,
-		 const char *allocator, float alloc_factor);
+		 const char *allocator, float alloc_factor,
+		 memtx_on_indexes_built_cb cb);
```

Comment:
Nit: Better name the argument `on_indexes_built`, because we may add other callbacks in future.

Reply:
Ok:
```diff
diff --git a/src/box/memtx_engine.cc b/src/box/memtx_engine.cc
index bea6edda7..760f7c1ed 100644
--- a/src/box/memtx_engine.cc
+++ b/src/box/memtx_engine.cc
@@ -1263,7 +1263,7 @@ memtx_engine_new(const char *snap_dirname, bool force_recovery,
                 uint64_t tuple_arena_max_size, uint32_t objsize_min,
                 bool dontdump, unsigned granularity,
                 const char *allocator, float alloc_factor,
-                memtx_on_indexes_built_cb cb)
+                memtx_on_indexes_built_cb on_indexes_built)
 {
        int64_t snap_signature;
        struct memtx_engine *memtx =
@@ -1370,7 +1370,7 @@ memtx_engine_new(const char *snap_dirname, bool force_recovery,
        }
        tuple_format_ref(memtx->func_key_format);
 
-       memtx->on_indexes_built_cb = cb;
+       memtx->on_indexes_built_cb = on_indexes_built;
 
        fiber_start(memtx->gc_fiber, memtx);
        return memtx;
diff --git a/src/box/memtx_engine.h b/src/box/memtx_engine.h
index 8f259c5ca..76e6acec7 100644
--- a/src/box/memtx_engine.h
+++ b/src/box/memtx_engine.h
@@ -218,7 +218,7 @@ memtx_engine_new(const char *snap_dirname, bool force_recovery,
                 uint64_t tuple_arena_max_size, uint32_t objsize_min,
                 bool dontdump, unsigned granularity,
                 const char *allocator, float alloc_factor,
-                memtx_on_indexes_built_cb cb);
+                memtx_on_indexes_built_cb on_indexes_built);
 
 int
 memtx_engine_recover_snapshot(struct memtx_engine *memtx,
@@ -351,12 +351,13 @@ memtx_engine_new_xc(const char *snap_dirname, bool force_recovery,
                    uint64_t tuple_arena_max_size, uint32_t objsize_min,
                    bool dontdump, unsigned granularity,
                    const char *allocator, float alloc_factor,
-                   memtx_on_indexes_built_cb cb)
+                   memtx_on_indexes_built_cb on_indexes_built)
 {
        struct memtx_engine *memtx;
        memtx = memtx_engine_new(snap_dirname, force_recovery,
                                 tuple_arena_max_size, objsize_min, dontdump,
-                                granularity, allocator, alloc_factor, cb);
+                                granularity, allocator, alloc_factor,
+                                on_indexes_built);
        if (memtx == NULL)
                diag_raise();
        return memtx;
```